java/ecs/fargate-load-balanced-service/src/main/java/com/amazonaws/cdk/examples/ECSFargateLoadBalancedApp.java

```diff
@@ -0,0 +1,14 @@
+package com.amazonaws.cdk.examples;
+
+import software.amazon.awscdk.core.App;
+
+public class ECSFargateLoadBalancedApp {
```

Comment:
Nit: I might remove the `ECS` prefix here (and elsewhere) for clarity, so e.g. `FargateLoadbalancedApp`.